src/ray/raylet/cgroup_setup.cc

```diff
@@ -0,0 +1,89 @@
+// Copyright 2024 The Ray Authors.
```

Comment:
nit: let's create a `src/ray/common/cgroup` folder and put all cgroup related things there?

If it's single file then we can also do `src/ray/common/cgroup_util.h|cc`

The reason is that it's not just raylet, gcs and everything else will also be inside cgroup.

Reply:
Updated.